[INTERNAL] Troubleshooting: Describe corrupt framework dependencies (#489)
SAP/ui5-tooling: docs/pages/Troubleshooting.md

## Issues Not Listed Here
Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#report-an-issue) on how to report an issue.

## UI5 Project
### Corrupt Framework Dependencies

When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.

> These issues can be the result of an aborted install during a preceding UI5 Tooling execution. Because of [a bug](https://github.com/SAP/ui5-tooling/issues/478), that has since been fixed, UI5 Tooling assumed that the preceding install was successful and work with the potentially corrupted dependency.

Reviewer: at the end of the sentence: "...and worked ..." or "... and works ..."

Author: Used the suggestion from @KlattG 👍

> These issues can be the result of an aborted install during a preceding UI5 Tooling execution. Because of [a bug](https://github.com/SAP/ui5-tooling/issues/478), that has since been fixed, UI5 Tooling assumed that the preceding install was successful and work with the potentially corrupted dependency.

Reviewer 2: ```suggestion
These issues can be the result of an aborted install during a preceding UI5 Tooling execution. Because of [a bug](https://github.com/SAP/ui5-tooling/issues/478) that has since been fixed, UI5 Tooling assumes that the preceding install was successful and uses the potentially corrupted dependency.
```

#### Resolution

> Remove the `.ui5/framework/` directory, located in your user's home directory.

Reviewer 2: ```suggestion
Remove the `.ui5/framework/` directory from your user's home directory.
```

```sh
rm -rf ~/.ui5/framework/
```

Any missing framework dependencies will be downloaded again during the next UI5 Tooling invocation.

### `~/.ui5` Taking too Much Disk Space

There are possibly many versions of UI5 framework dependencies installed on your system, taking a large amount of disk space.

#### Resolution

> Remove the `.ui5/framework/` directory, located in your user's home directory:

Reviewer 2: ```suggestion
Remove the `.ui5/framework/` directory from your user's home directory:
```
